package.json

```diff
@@ -7,8 +7,8 @@
   ],
   "packageManager": "pnpm@7.20.0",
   "engines": {
-    "node": ">=18.x",
-    "pnpm": ">=7.0.0"
+    "node": ">=v18.12.1",
+    "pnpm": ">=7.20.0"
```

Comment:
This is a rather very strict policy, are we sure we need this ?

Reply:
This is what we have in all our other projects. We also already set the `packageManager` field to `pnpm@7.20.0`, so if we want to allow broader versions we should also update it.

Reply:
Update: https://github.com/scaleway/scaleway-ui/pull/2161#issuecomment-1369775010